pip/flatpak-pip-generator

```diff
@@ -116,14 +116,19 @@ for package in packages:
             '--dest',
             tempdir
         ]
+
+        name_for_pip = package.name
+        if package.vcs:
+         name_for_pip = '.'
```

Comment:
Bad whitespace.

Reply:
Good catch, fixed in a force push to have a single commit.